src/emails/templates/example/index.mjml

```diff
@@ -0,0 +1,18 @@
+<mjml>
+  <mj-body>
+    <mj-section>
+      <mj-column>
+        <mj-text>
+          <!-- Rendering a Fluent string: -->
+          <h1>{data-breach-header-1}</h1>
+          <!-- Rendering a variable directly, without going through Fluent: -->
+          <p>Hello {subscriberName},</p>
```

Comment:
Showing this unlocalised string concatenation in the example is probably a bad idea, as it suggests that doing this would be ok.